macros/edr/tests/test_utils/collect_table_metrics.sql

```diff
@@ -18,42 +18,39 @@
     {% for metric in table_metrics %}
         {% do metric_names.append(metric.name) %}
     {% endfor %}
-    {% set table_monitors = elementary.get_final_table_monitors(monitors=metric_names) %}
-    {% if not table_monitors %}
-        {% do return(none) %}
-    {% endif %}
-
-    {% if dimensions and table_monitors != ["row_count"] %}
-        {% do exceptions.raise_compiler_error("collect_metrics test does not support non row_count dimensional table metrics.") %}
-    {% endif %}
```

Comment:
Why did you remove this? Seems still to be a relevant check (that we support dimensions only for row count)

Reply:
Was just moved below.

Reply:
I see below that you just check if there is one metric, but not that its type is row_count

Reply:
https://github.com/elementary-data/dbt-data-reliability/pull/736/commits/12a7cc97e3210a9e0d2feaf6d5ff2af7e1643741